troop/forms.py

```diff
@@ -47,15 +41,53 @@ class Meta:
             "comment",
         ]
         widgets = {
-            "troop": HiddenInput,
-            "gender": RadioSelect,
-            "birthday": DateInput(format="%Y-%m-%d", attrs={"type": "date"}),
-            "age_section": RadioSelect(attrs={"required": True}),
-            "attendance": CheckboxSelectMultiple,
-            "diet": CheckboxSelectMultiple,
-            "comment": Textarea(attrs={"cols": 32, "rows": 4}),
+            "troop": forms.HiddenInput,
+            "gender": forms.RadioSelect,
+            "birthday": forms.DateInput(format="%Y-%m-%d", attrs={"type": "date"}),
+            "age_section": forms.RadioSelect(attrs={"required": True}),
+            "attendance": forms.CheckboxSelectMultiple,
+            "diet": forms.CheckboxSelectMultiple,
+            "comment": forms.Textarea(attrs={"cols": 32, "rows": 4}),
         }
 
 
-class NamiSearchForm(Form):
+class NamiSearchForm(forms.Form):
     nami = Participant._meta.get_field("nami").formfield()
+
+
+class SendEmailForm(forms.Form):
+    SECTION_CHOICES = (
+        ("all", _("All sections")),
+        (Participant.SECTION_BEAVER, _("Beavers")),
+        (Participant.SECTION_CUB, _("Cubs")),
+        (Participant.SECTION_SCOUT, _("Scouts")),
+        (Participant.SECTION_VENTURER, _("Venturers")),
+        (Participant.SECTION_ROVER, _("Rovers")),
+        ("no", _("No section")),
+    )
+
+    STATUS_CHOICES = (
+        ("all", _("All")),
+        ("lea", _("Only leaders")),
+        ("chi", _("Only children")),
+    )
+
+    sender = forms.CharField(label=_("Sender"), disabled=True)
+    reply_to = forms.CharField(label=_("Reply to"))
+
+    section = forms.ChoiceField(label=_("Section"), choices=SECTION_CHOICES)
+    status = forms.ChoiceField(label=_("Status"), choices=STATUS_CHOICES)
+
+    subject = forms.CharField(label=_("Subject"))
+    message = forms.CharField(
+        label=_("Message"), widget=forms.Textarea(attrs={"cols": 32, "rows": 4})
+    )
+
+    def __init__(self, user, troop, *args, **kwargs):
+        super(SendEmailForm, self).__init__(*args, **kwargs)
+
+        if not self.initial:
+            self.initial["sender"] = "{} {} <{}>".format(
+                user.first_name, user.last_name, settings.DO_NOT_REPLY_EMAIL
+            )
+            self.initial["reply_to"] = user.email
```

Comment:
```suggestion
            self.initial["reply_to"] = user.email
            self.initial["section"] = "all"
```

Reply:
"all" is already the default since it is the first option. I wouldn't add this line, ok?